### DISCUSSION TITLE
Long Overdue Thanks for Keeping this Stellar Feed Reader Alive

### DISCUSSION THREAD

**Discussion by bongochong** [Xyrio/RSSOwlnix#122] [General]
I just wanted to offer my heartfelt thanks and appreciation for your work on keeping RSSOwl alive and well. I had switched to QuiteRSS a few months after the original RSSOwl's development ceased, and thought the project was dead and gone. While QuiteRSS is solid, it's just not as flexible or responsive as RSSOwl, and I found myself missing it every time I'd go to check my feed subscriptions. Around a year ago I discovered that you had taken it upon yourself to revive and continue maintaining the project. I switched to RSSOwl-nix in a heartbeat and am back to actually enjoying my daily routine of sifting through various news sources on my PCs.

While this knocks other desktop feed readers out of the park IMHO, I do have one - and only one - criticism (which also applied to the original RSSOwl): The embedded web browser would be greatly improved if its fonts were configurable. I'd love to know if said feature is feasible to implement.

Either way, many thanks!

**Comment by Xyrio** [comment]
i dont intent to touch the internal browser. only maybe to remove it (but that would be some work too). for security reasons (uses IE11 on windows) it is better to use an external one that is kept up to date.

**Reply by bongochong** [reply]
I use the embedded browser with JS off to view articles which RSSOwl can't fetch the full content for, and only open articles in an external browser if they fail to render without JS. I feel you though, and thanks again!

**Reply by Xyrio** [reply]
fyi checkout uMatrix plugin for browsers if you want to control what is loaded/executed. (not made by me)

**Reply by bongochong** [reply]
I'm quite familiar with uMatrix, but I prefer uBO, and I maintain filter lists for it. My only issue is with the lack of font choices in RSSOwl's embedded browser, not my configuration of other web browsers. Though you don't intend to work on the browser component, I will still continue to use your fork of RSSOwl, as it is a highly configurable and responsive feed reader. I thank you for your continued work on it.

**Reply by slemay2** [reply]
I too agree - excellent and still the best RSS reader. Any chance you'll be putting out a new update/beta sometime soon? It's been a year since the last. Thanks again.

**Reply by Xyrio** [reply]
yea im working on one. but im stuck on some problems currently. started in march to updated to latest eclipse rcp. nothing fancy but should open up more possibilites for further fixes/improvements. but the 32bit version is not supported anymore by eclipse.
